src/lcs.nim

```diff
@@ -29,18 +28,23 @@ proc runMain() =
   if licenseType == "":
     # exit cli if type is not set
     echo "License type flag required! (--type, -t)"
-    quit(0)
+    quit(1)
```

Comment:
It's generally better to fail with a non-zero exit code if the program didn't actually complete.